openmetadata-ui/src/main/resources/ui/src/components/AddService/AddService.component.tsx

```diff
@@ -117,38 +112,13 @@ const AddService = ({
 
   // Configure service name
   const handleConfigureServiceBackClick = () => setActiveServiceStep(1);
-  const handleConfigureServiceNextClick = (descriptionValue: string) => {
-    setDescription(descriptionValue.trim());
-
-    if (!serviceName.trim()) {
-      setShowErrorMessage({ ...showErrorMessage, name: true, isError: true });
-    } else if (nameWithSpace.test(serviceName)) {
-      setShowErrorMessage({
-        ...showErrorMessage,
-        nameWithSpace: true,
-        isError: true,
-      });
-    } else if (delimiterRegex.test(serviceName)) {
-      setShowErrorMessage({
-        ...showErrorMessage,
-        delimit: true,
-        isError: true,
-      });
-    } else if (!isUrlFriendlyName(serviceName.trim())) {
-      setShowErrorMessage({
-        ...showErrorMessage,
-        specialChar: true,
-        isError: true,
-      });
-    } else if (serviceName.length < 1 || serviceName.length > 128) {
-      setShowErrorMessage({
-        ...showErrorMessage,
-        nameLength: true,
-        isError: true,
-      });
-    } else if (!showErrorMessage.isError) {
-      setActiveServiceStep(3);
-    }
+  const handleConfigureServiceNextClick = (
+    serviceName: string,
+    description: string
+  ) => {
+    setDescription(description);
+    setServiceName(serviceName);
```

Comment:
Can we reduce it to single state variable?

Reply:
Yes, I will make the change.